docs/source/install/k8s_ha.rst

```diff
@@ -46,15 +46,55 @@ makes installing the complex StackStorm infrastructure as easy as:
   # Add Helm StackStorm repository
   helm repo add stackstorm https://helm.stackstorm.com/
 
-  helm install stackstorm/stackstorm-ha
+  # Update the StackStorm repository
+  helm repo update stackstorm
 
-Once the deployment is finished, it will show you the first steps to get started working with the new cluster via WebUI
-or ``st2`` CLI client:
+  # Install StackStorm HA with an automatically-generated release name in the "stackstorm" namespace
+  # (Replace "--generate-name" with a release name if you'd like to name the deployment, omit the
+  # "--namespace stackstorm" and "--create-namespace" flags if you'd like to deploy to the "default"
+  # namespace)
+  helm install --generate-name --namespace stackstorm --create-namespace stackstorm/stackstorm-ha
```

Comment:
I don't like the namespace and create namespace flags in the same command. Perhaps you can keep one and give the other as an option in the comments on the top?